src/Microsoft.Framework.DesignTimeHost/ApplicationContext.cs

```diff
@@ -1233,9 +1233,11 @@ private static string GetProjectRelativeFullPath(Project referencedProject, stri
 
         private static DependencyDescription CreateDependencyDescription(LibraryDescription library)
         {
+            var name = library.Identity.Name;
+            //var name = LibraryRange.GetAssemblyName(library.Identity.Name);
             return new DependencyDescription
             {
-                Name = library.Identity.Name,
```

Comment:
Something will change. Once I can get it to build I'm going to be uncommenting line 1237. I left it commented out because I want to see exactly what DTH does without it (it should start showing "fx/System.IO" and friends on the References node, but we want to trim it back to "System.IO"). I don't want to apply the fix until I see the issue though :).

If it ends up being unnecessary, I'll revert the whole file :)